bundle/manifests/sailoperator.io_istiocnis.yaml

```diff
@@ -1512,6 +1512,9 @@ spec:
                 description: Reports the current state of the object.
                 type: string
             type: object
+        required:
+        - metadata
+        - spec
```

Comment:
aren't these changes that would require a version bump?

Reply:
I initially thought that both of these can actually be required and since all existing objects stored in the API should have these fields, we can make them required. However, I've now checked the core k8s kinds and they all mark these two as optional, so I've now done the same.